qiime2/sdk/action.py

```diff
@@ -283,45 +284,54 @@ def _build_annotations(self):
         return annotations
 
     def _build_numpydoc(self):
-        numpydoc = ""
-        numpydoc += "{}\n\n".format(self.name)
-        numpydoc += "{}\n\n".format(self.description)
+        numpydoc = []
+        numpydoc.append(textwrap.fill(self.name, width=79))
+        numpydoc.append(textwrap.fill(self.description, width=79))
 
         sig = self.signature
         params = True if len(sig.inputs) != 0 else False
```

Comment:
Missed this earlier, but this is just `bool(sig.inputs)`. In Python things that can be described as empty are also "falsey" (technical term) which means that in the context of a boolean expression they will be evaluated with respect to their truthy-ness and falsey-ness.